dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -31,6 +31,8 @@ namespace ToastABI
     using namespace ::ABI::Microsoft::Internal::ToastNotifications;
 }
 
+constexpr PCWSTR defaultAppNotificationIcon = LR"(ms-resource://Windows.UI.ShellCommon/Files/Images/DefaultSystemNotification.png)";
```

Comment:
Is this resource guaranteed in all downlevel OS builds?

Reply:
Yes, Lester confirmed it.

Reply:
... but it puts now a reverse dependency on the `Windows.UI.Shellcommon` component to host this forever. Is this a publicly documented value? Can you link to the `docs.microsoft.com` topic about how to use it?

Reply:
I don't see this icon being publicly documented.

Reply:
@DrusTheAxe Can we get this icon installed as part of the Framework Package? That might solve the reverse dependency issue ?

Reply:
There are valid concerns in this thread. I have started an email thread to address this offline.

Reply:
2 options here:
1) Add a UDK API since this is closed source and can change over time. Have a contract in place with Shell to handle name changes in the future or prevent name changes in the future.
2) Include the icons in Framework SDK and infer to those icons instead.